apps-rendering/src/components/Callout/calloutForm.tsx

```diff
@@ -0,0 +1,186 @@
+import type { FormField as FormFieldType } from '@guardian/apps-rendering-api-models/formField';
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	Button,
+	InlineError,
+	InlineSuccess,
+	Link,
+} from '@guardian/source-react-components';
+import { useState } from 'react';
+import type { FC } from 'react';
+import { ContactText, Disclaimer, FormField } from './formFields';
+import { ShareLink } from './shareLink';
+import { calloutForm, calloutSubmitButton } from './styles';
+
+export interface CalloutFormProps {
+	format: ArticleFormat;
+	id: number;
+	fields: FormFieldType[];
+	disableInputs?: boolean;
```

Comment:
doesn't look like we're using this prop - can we remove?

Reply:
Great spot!